composer.json

```diff
@@ -110,7 +110,8 @@
             "stubs",
             "rules-tests/CodingStyle/Rector/Namespace_/ImportFullyQualifiedNamesRector/Source",
             "rules-tests/Renaming/Rector/Name/RenameClassRector/Source",
-            "rules-tests/Renaming/Rector/FileWithoutNamespace/PseudoNamespaceToNamespaceRector/Source"
+            "rules-tests/Renaming/Rector/FileWithoutNamespace/PseudoNamespaceToNamespaceRector/Source",
+            "rules-tests/TypeDeclaration/Rector/ClassMethod/AddReturnTypeDeclarationBasedOnParentClassMethodRector/Source"
```

Comment:
This should not be needed, as the classes mist be psr-4 here

Reply:
See discussion above. Without it PHPStan fails in CI. @samsonasik suggested this as fix and it worked.

Reply:
Not sure why it is needed because it seems more like weird PHPStan bug to me. Any idea how it could be solved differently?